Sockethead.Razor/PRG/ModelStateHelpers.cs

```diff
@@ -36,6 +37,8 @@ public static ModelStateDictionary DeserializeModelState(string serialisedErrorL
 
             foreach (var item in errorList)
             {
+                item.RawValue = item.RawValue is JArray jArray ? jArray.ToObject<object[]>() : item.RawValue;
+                
```

Comment:
I love the conditional operator as much as the next guy, but isn't this clearer?

`if (item.RawValue is JArray jArray)
    item.RawValue = jArray.ToObject<object[]>();`